app/assets/stylesheets/expenses.css.scss

```diff
@@ -0,0 +1,7 @@
+#create_expense_link {
+  float:right;
+}
+
+#create_expense_link_text {
```

Comment:
Avoid using id selectors<br>Selector `create_expense_link_text` should be written in lowercase with hyphens